src/content/developers/docs/data-and-analytics/block-explorers/index.md

```diff
@@ -178,12 +178,12 @@ Slots are opportunities for block creation, the data available for each slot inc
 - Status – The status of the slot (Proposed/Missed)
 - Time – The slot timestamp
 - Proposer – The validator that proposed the block for the slot
-- Block root – ?
+- Block root – The hash-tree-root of the BeaconBlock
 - Parent root – The hash of the block that came before
-- State root – ?
+- State root – The hash-tree-root of the BeaconState
 - Signature – ?
 - Randao reveal – ?
-- Graffiti
+- Graffiti – A block proposer can include 32 byte long message to its block proposal
```

Comment:
(small grammar suggestion)
- Graffiti – A block proposer can include a 32-byte long message to its block proposal

Reply:
Thanks!